cypress/e2e/calculator.cy.js

```diff
@@ -0,0 +1,100 @@
+const clickDigit = (text) => cy.contains(".digit", text).click();
+const clickOperator = (text) => cy.contains(".operation", text).click();
+const clickAllClear = () => cy.get(".modifier").click();
+const getDisplayNumber = (text) => cy.get("#total").should("have.text", text);
+
+describe("사칙연산 테스트", () => {
+	it("22 + 3 = 25", () => {
+		cy.visit("http://127.0.0.1:5500/index.html");
+
```

Comment:
```suggestion
describe("사칙연산 테스트", () => {
	beforeEach(() => {
	  cy.visit("http://127.0.0.1:5500/index.html");
	})
	
	it("22 + 3 = 25", () => {
```
똑같은 페이지를 방문한다면 이렇게 바꿔볼 수 있을거같아요

Reply:
몰랐던 기능을 알아갑니다! 수정했습니다